common/modules/utils/src/lib.rs

```diff
@@ -25,7 +26,7 @@ pub trait UtilModule {
 
     fn get_non_empty_payments(&self) -> PaymentsVec<Self::Api> {
         let payments = self.call_value().all_esdt_transfers();
-        require!(!payments.is_empty(), NO_PAYMENT_ERR_MSG);
+        require!(!payments.is_empty(), "No payments");
```

Comment:
Since it's not really used in multiple places (at least not for anyone importing this module and using this function), I'd say it's fine as a plain string.